### PR TITLE
Cleanup partest.TestKinds

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

File: src/partest/scala/tools/partest/TestKinds.scala [hunk 1]
```diff
@@ -13,24 +13,9 @@ object TestKinds {
...
-    // (srcDir relativize p.toCanonical).segments match {
-    //   case (".." :: "scaladoc" :: xs) => xs.head
-    //   case xs => xs.head
-    // }
-  }
-  def logOf(p: Path) = {
-    p.parent / s"${p.stripExtension}-${kindOf(p)}.log"
-    // p.parent / s"${p.stripExtension}.log"
-  }
+  def kindOf(p: Path) = p.toAbsolute.segments takeRight 2 head
```
Comment (som-snytt) [inline src/partest/scala/tools/partest/TestKinds.scala]: b/c my recent work on postfix, could you normalize it? i.e. fully dotted and parens?

**Comment by SethTisue** [comment]
thank you, Dale!

**Comment by som-snytt** [comment]
Thanks for not minding my comment. Or you did mind it, in the sense of mind the gap.

**Comment by dwijnand** [comment]
It's ok. I think postfix syntax looks nice, but it's clunky with the current semicolon inference.

**Comment by som-snytt** [comment]
@dwijnand it's a mission. https://github.com/scala/scala/pull/6663#issuecomment-390460384

**Pull request merged**